src/components/AvailablePlatforms.jsx

```diff
@@ -1,27 +1,103 @@
-import React from 'react'
-import Card from 'cozy-ui/transpiled/react/Card'
-import Icon from 'cozy-ui/transpiled/react/Icon'
+import React, { useState, useCallback } from 'react'
 import { ButtonLink } from 'cozy-ui/transpiled/react/Button'
 import supportedPlatforms from 'supportedPlatforms'
 import Stack from 'cozy-ui/transpiled/react/Stack'
-import { Text } from 'cozy-ui/transpiled/react/Text'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import { isAndroid, isIOS } from 'cozy-device-helper'
+
+import Card from 'cozy-ui/transpiled/react/Card'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import { Text } from 'cozy-ui/transpiled/react/Text'
+import Modal, { ModalContent } from 'cozy-ui/transpiled/react/Modal'
+
+import playBadgeEn from 'assets/google-play-badge-en.png'
+import playBadgeFr from 'assets/google-play-badge-fr.png'
+
+import appStoreBadgeEn from 'assets/app-store-badge-en.png'
+import appStoreBadgeFr from 'assets/app-store-badge-fr.png'
+import flag from 'cozy-flags'
+
+const playStoreBadges = {
+  en: playBadgeEn,
+  fr: playBadgeFr
+}
+
+const appStoreBadges = {
+  en: appStoreBadgeEn,
+  fr: appStoreBadgeFr
+}
 
 const PlatformButton = props => {
   const { icon, ...rest } = props
+  const color = props.theme == 'primary' ? 'var(--white)' : 'var(--slateGrey)'
   return (
     <ButtonLink
-      {...rest}
-      icon={<Icon icon={icon} size={16} color="var(--slateGrey)" />}
+      icon={<Icon icon={icon} size={16} color={color} />}
       theme="secondary"
       className="u-mb-half"
+      {...rest}
     />
   )
 }
 
+const AppStoreButton = props => {
+  const { lang } = useI18n()
+  const badge = appStoreBadges[lang] || appStoreBadges.en
+  return (
+    <a {...props}>
+      <img src={badge} style={{ height: '3rem', marginRight: '0.5rem' }} />
+    </a>
+  )
+}
+
+const PlayStoreButton = props => {
+  const { lang } = useI18n()
+  const badge = playStoreBadges[lang] || playStoreBadges.en
+  return (
+    <a {...props}>
+      <img src={badge} style={{ height: '3rem' }} />
+    </a>
+  )
+}
+
+export const InstallNativeAppButton = props => {
+  const [isSmartphoneModalOpened, setSmartphoneModalOpened] = useState(false)
+  const mobileOS = isAndroid() ? 'android' : isIOS() ? 'ios' : null
+  const handleOpenModal = useCallback(() => {
+    setSmartphoneModalOpened(true)
+  }, [setSmartphoneModalOpened])
+
+  const handleDismissModal = useCallback(() => {
+    setSmartphoneModalOpened(false)
+  }, [setSmartphoneModalOpened])
+
+  return (
+    <>
+      <PlatformButton
+        icon="phone"
+        href={mobileOS !== null ? storeLinksPerOS[mobileOS] : null}
+        onClick={mobileOS === null ? handleOpenModal : null}
+        {...props}
+      />
+      {isSmartphoneModalOpened ? (
+        <Modal size="small" dismissAction={handleDismissModal}>
```

Comment:
Since we're on Mobile, you should use ExperimentalDialog now.

Reply:
We're not always on mobile here, is ExperimentalDialog ok for desktop also ?

Reply:
I guess if we want those dialogs to be used, we have to rename them to Dialog and have a migration route, I am still unsure what's the plan for them.

Reply:
They are still experimental because joel has to work on a more precise spec. So they are not finished yet, and for that reason they are still experimental and we do not provide a migration path yet. But we expect to use ExpDialog when we write a new code.

Is it ok for you? Can we act that?

Reply:
I am unclear as to what is experimental, is it the design ? Do we think there will be changes to the API ? I'll change my PR to use them.

Reply:
Fixed in https://github.com/cozy/cozy-passwords/pull/21/commits/142d35b9e4f75a7f35c8ac09ce31030d70e0a16d